docs/sdkx_ios/outbound-support.mdx

```diff
@@ -89,114 +86,58 @@ To pass the outbound support data to Helpshift, follow these steps -
 
 </Admonition>
 
-For example -
-_Following code shows how to handle incoming push notification and posting it on the notification bar of the device_
+For example, following code shows how to handle incoming push notification and post it on the notification bar of the device -
 
-<Tabs>
+<Tabs groupId="ios-languages">
 
 <TabItem value="Objective-C" label="Objective-C">
 
 ```objc
-
-    - (void) userNotificationCenter:(UNUserNotificationCenter *)center willPresentNotification:(UNNotification *)notification withCompletionHandler:(void (^)(UNNotificationPresentationOptions)) completionHandler  API_AVAILABLE(ios(10.0)){
-        //Helpshift normal notification.
-        if([[notification.request.content.userInfo objectForKey:@"origin"] isEqualToString:@"helpshift"]) {
-            NSLog(@"userNotificationCenter:willPresentNotification for helpshift origin");
-            [Helpshift handleNotificationWithUserInfoDictionary:notification.request.content.userInfo isAppLaunch:NO withController:self.window.rootViewController];
+- (void) userNotificationCenter:(UNUserNotificationCenter *)center
+    willPresentNotification:(UNNotification *)notification
+    withCompletionHandler:(void (^)(UNNotificationPresentationOptions))completionHandler {
+    if([[notification.request.content.userInfo objectForKey:@"origin"] isEqualToString:@"helpshift"]) {
+        // This is a notification from Helpshift push notification system
+        [Helpshift handleNotificationWithUserInfoDictionary:notification.request.content.userInfo isAppLaunch:NO withController:self.window.rootViewController];
+        completionHandler(UNNotificationPresentationOptionNone);
+    } else {
+        NSString* proactiveLink = [[notification.request.content.userInfo objectForKey:@"aps"] objectForKey:@"helpshift_proactive_link"];
```

Comment:
Can we update this example, currently we are suggesting reading the data from "aps" object which is not the recomded. if users are using additional package something aps object will not be passed, because aps has info on notification sound, type etc.. 

Checked the apple website they suggest to put custom data outside of the aps object. 
https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification#2943359